docs/Theming.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: "Theming"
+title: 'Theming'
```

Comment:
auto formatting of markdown files again...

Reply:
Damn it